plugins/source/aws/resources/services/autoscalingplans/plans.go

```diff
@@ -0,0 +1,52 @@
+package autoscalingplans
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/autoscalingplans"
+	"github.com/aws/aws-sdk-go-v2/service/autoscalingplans/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/transformers"
+)
+
+func Plans() *schema.Table {
+	tableName := "aws_autoscaling_plans"
+	return &schema.Table{
+		Name:        tableName,
+		Description: `https://docs.aws.amazon.com/autoscaling/plans/APIReference/API_ScalingPlan.html`,
+		Resolver:    fetchPlans,
+		Multiplex:   client.ServiceAccountRegionMultiplexer(tableName, "autoscaling-plans"),
```

Comment:
This is `ServiceAccountRegionNamespaceMultiplexer` in other autoscaling resources, but no namespace inputs here.